composer.json

```diff
@@ -26,12 +26,12 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.1"
+        "php": "^7.1|^8.0"
```

Comment:
It seems that single pipe `|` is considered [deprecated](https://github.com/composer/composer/issues/6755), could you please use `||` instead for version ranging?

Reply:
Done